common/production_methods/babylon_urban.txt

```diff
@@ -1,44 +1,256 @@
-﻿pm_public_roads = {
+﻿
+##Public Transport
+
+pm_rickshaws = {
 	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
 	unlocking_technologies = {
-		paved_roads
+		urbanization
 	}
 	state_modifiers = {
 		workforce_scaled = {
-			state_pollution_generation_add = 50
+			state_pollution_generation_add = 0
 		}
 	}
-	country_modifiers = {
-		unscaled = {
-			state_migration_pull_mult = 0.1
+
+	building_modifiers = {
+		workforce_scaled = {
+			goods_input_fabric_add = 2
+			goods_input_wood_add = 4
+			# output goods
+			goods_output_transportation_add = 10
+			
+			#profit = (10*30)-(6*20) = 200*52 = 10,400
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = 2500
+		}
+	}
+}
+
+pm_stagecoaches = {
+	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		urbanization
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_pollution_generation_add = 0
 		}
 	}
 
 	building_modifiers = {
 		workforce_scaled = {
-			goods_input_automobiles_add = 1
+			goods_input_fabric_add = 4
+			goods_input_wood_add = 4
+			goods_input_grain_add = 4
+			# output goods
+			goods_output_transportation_add = 10
+			
+			#profit = (10*30)-(12*20) = 60*52 = 3120 
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = 1000
+		}
+	}
+}
+
+pm_trams = {
+	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		railways
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_pollution_generation_add = 10
+		}
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			goods_input_engines_add = 2
+			goods_input_coal_add = 5
+			goods_output_transportation_add = 15
+			
+			#profit = (15*30)-((2*60)+(5*30)) = 180 = 9360
+		}
+
+		level_scaled = {
+			building_employment_engineers_add = 100
+			building_employment_machinists_add = 400
+			building_employment_laborers_add = 1000
+		}
+	}
+}
+
+pm_steam_buses = {
+	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		threshing_machine
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_pollution_generation_add = 25
+		}
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			goods_input_coal_add = 5
+			goods_input_engines_add = 5
+			# output goods
+			goods_output_transportation_add = 20
+			
+			#profit = (20*30)-((5*30)+(5*60)) = 150*52 = 7800
```

Comment:
So this is the reason you do this calculation. Generally profitability should never decrease when going up in tech, yet here it does.